templates/named.conf.options.erb

```diff
@@ -1,5 +1,5 @@
 options {
-	directory "/var/cache/bind";
+	directory "<%= scope.lookupvar('::dns::server::params::data_dir') %>";
```

Comment:
My apologies! I thought Paulo's comment was on a different PR.  I should make my replies on github not through email.